src/core/config/npo/snapshots/tailcall__core__config__npo__tracker__tests__multiple_type_usage.snap

```diff
@@ -0,0 +1,8 @@
+---
+source: src/core/config/npo/tracker.rs
+expression: actual
+---
+query { t2 { t2 { n } } }
+query { t2_ls { t2 } }
+query { t3 { t3 { n } } }
+query { t3_ls { t3 { n } } }
```

Comment:
on previous version it was 

```
query { t2 { t2 { n } } }
query { t2 { t2 { n } } }
query { t2_ls { t2 } }
query { t2 { t2 { n } } }
```

because of caching the whole paths